.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ executors:
 aliases:
   - &docs-job
     docker:
-      - image: sceptreorg/sceptre-circleci:2.0.0
```

Comment:
What about using the `cimg/python:3.12` image, which tracks the Python feature version, rather than `cimg/python:3.12.0`, which tracks the bugfix version?

[The push history of the `3.11` tags](https://hub.docker.com/r/cimg/python/tags?page=1&name=3.11&ordering=last_updated) shows that `3.11` and `3.11.6` have the same digest.

Reply:
Doesn't it need the `node` variant?

Reply:
> Doesn't it need the `node` variant?

All of the tests passed so i guess not.